### ISSUE TITLE
How long will the verkko take to finish the HG002 genome assembly?

### ISSUE THREAD

**Issue by bioinformaticspcj** [marbl/verkko#89]
Dear the authors,

Thanks for developing such a wonderful program. In your paper published on bioRxiv (https://doi.org/10.1101/2022.06.24.497523), you said you assembled the HG002 genome in good quality. However you did not tell the computing resource consumption (such as the number of CPUs used) as well as the time  consumption for running the program. I am wondering if my server could be power enough to run verkko. Could you provide me with these information?

Thanks a lot
Best,
Bob

**Comment by brianwalenz** [comment]
See Supplementary Table 1 https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2022.06.24.497523/DC2/embed/media-2.xlsx.  For human-size assemblies, wall time will be _approximately_ 36 + CPU_hours / number_of_CPUs.

**Comment by bioinformaticspcj** [comment]
Dear brianwalenz and skoren,


Thanks for your timely reply. I just downloaded and installed the 1.0.16 version of GraphAligner. Does this version of GraphAligner have been speed improvements or I have to wait for the new version of the program to use?


Best,
Bob



------------------ 原始邮件 ------------------
发件人:                                                                                                                        "marbl/verkko"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年7月20日(星期三) 晚上9:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [marbl/verkko] How long will the verkko take to finish the HG002 genome assembly? (Issue #89)





 
I'll also add that the times will likely be about half of this since there have been speed improvements to GraphAligner since the paper.

**Comment by skoren** [comment]
Yes, that's the version corresponding to Verkko's v1.0 release (https://github.com/marbl/verkko/releases/tag/v1.0) which you can see has the GraphAligner speedup in the change log.
